recipes/cp2k/Linux-x86-64-conda.sopt

```diff
@@ -0,0 +1,32 @@
+# Tested with: GFortran 7.4.0, LAPACK 3.8.0, FFTW 3.3, Libint 1.1.6, Libxc 4.3.4, libxsmm 1.12
+# Author: Matthias Krack (matthias.krack@psi.ch, PSI, June 2019)
+CC          = $(CC)
+CPP         =
+FC          = $(FC)
+LD          = $(FC)
+AR          = ar -r
```

Comment:
I think it should be
```suggestion
AR          = $(AR) -r
```

Reply:
This would cause a recursive loop.

Reply:
How about `$(BUILD_PREFIX)/bin/x86_64-conda_cos6-linux-gnu-ar`